app/models/user_alternative_uuid.rb

```diff
@@ -0,0 +1,6 @@
+class UserAlternativeUuid < ActiveRecord::Base
```

Comment:
In another comment talked about maybe calling the trusted uuid an "external UUID" - if you like that, maybe change this to match ... `UserExternalUuid`